examples/loitering/README.md

```diff
@@ -0,0 +1,91 @@
+# Detect people loitering example
+
+## Setup
+
+You can follow the instructions [here](../../vqpy/README.md) to prepare your environment for VQPy. No other dependency is used.
+
+Video used in this query can be found [here](https://youtu.be/EuLMrUFNRxQ).
```

Comment:
mention that video need to be downloaded to local

Reply:
Added instruction to section "Run example"